<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>828</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Travelagency UpAndAway</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>971</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>128</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Buchungsnummer</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Preis</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Kunde</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Buchungsdetails</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Buchungsnummer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>von</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCalendarWidget" name="calendarWidgetFromDate"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>bis</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCalendarWidget" name="calendarWidgetToDate"/>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Preis</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="priceField"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Reise</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="travelField"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Kunde</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="customerField"/>
       </item>
       <item row="9" column="1">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="FlightDetailWindow">
          <widget class="QWidget" name="gridLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>371</width>
             <height>119</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="3" column="1">
             <widget class="QLineEdit" name="seatPrefDetailsField"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Zieldestination</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="toDestDetailsField"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Abflugdestination</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Sitzplatzpräferenz</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="fromDestDetailsField"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Airline</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="airlineDetailsField"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="HotelDetailsWindow">
          <widget class="QWidget" name="gridLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>371</width>
             <height>88</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Stadt</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Hotel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Raucherzimmer</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="hotelNameDetailField"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="cityNameDetailField"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="smokeDetailField"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="RentalCarReservationDetailsWindow">
          <widget class="QWidget" name="gridLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>341</width>
             <height>119</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Abholstation</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Versicherung</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Rückgabestation</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Autovermietung</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="CompanyDetailField"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="pickupDestinationDetailsField"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="returnLocationDetailsField"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="insuranceTypeDetailsField"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="searchbar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Suchen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunktionen">
    <property name="title">
     <string>Funktionen</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDatei_einlesen"/>
    <addaction name="actionBuchungen_anzeigen"/>
    <addaction name="separator"/>
    <addaction name="actionProgramm_beenden"/>
   </widget>
   <addaction name="menuFunktionen"/>
  </widget>
  <action name="actionDatei_einlesen">
   <property name="text">
    <string>Datei einlesen</string>
   </property>
  </action>
  <action name="actionBuchungen_anzeigen">
   <property name="text">
    <string>Buchungen anzeigen</string>
   </property>
  </action>
  <action name="actionProgramm_beenden">
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </action>
  <action name="actionKundensuche">
   <property name="text">
    <string>Kundensuche</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
